Source/WebCore/PAL/pal/crypto/win/CryptoDigestWin.cpp

```diff
@@ -80,11 +80,11 @@ std::unique_ptr<CryptoDigest> CryptoDigest::create(Algorithm algorithm)
     return nullptr;
 }
 
-void CryptoDigest::addBytes(const void* input, size_t length)
+void CryptoDigest::addBytes(std::span<const uint8_t> input)
 {
-    if (!input || !length)
+    if (!input.data() || !input.size())
```

Comment:
`if (input.empty())` should suffice here.

Reply:
Will fix.